hack/lib.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+readonly CONTROL_PLANE_SERVICE_ACCOUNT="cloud-run-events"
+readonly CONTROL_PLANE_NAMESPACE="cloud-run-events"
+
+readonly PUBSUB_SERVICE_ACCOUNT="cre-pubsub"
+
+readonly SERVICE_ACCOUNT_EMAIL_KEY="EMAIL"
+
+readonly ZONAL_CLUSTER_LOCATION_TYPE="zonal"
+readonly REGIONAL_CLUSTER_LOCATION_TYPE="regional"
+
+# Constants used for both init_XXX.sh and e2e-xxx.sh
+export K8S_CONTROLLER_SERVICE_ACCOUNT=controller
+export CONTROL_PLANE_SECRET_NAME="google-cloud-key"
+export PUBSUB_SECRET_NAME="google-cloud-key"
+
+function init_control_plane_service_account() {
+  local project_id=${1}
+  local control_plane_service_account=${2}
+
+  echo "parameter project_id used when initiating control plane service account is'${project_id}'"
+  echo "parameter control_plane_service_account used when initiating control plane service account is'${control_plane_service_account}'"
+
+  # Enable APIs.
+  gcloud services enable pubsub.googleapis.com
+  gcloud services enable storage-component.googleapis.com
+  gcloud services enable storage-api.googleapis.com
+  gcloud services enable cloudscheduler.googleapis.com
+  gcloud services enable cloudbuild.googleapis.com
+  gcloud services enable logging.googleapis.com
+  gcloud services enable stackdriver.googleapis.com
+  # Create the service account for the control plane if it doesn't exist
```

Comment:
nice! Please also remove the "TODO(#896)" in https://github.com/google/knative-gcp/blob/master/docs/install/install-knative-gcp.md.

Reply:
I will remove it when I work on updating the docs